### ISSUE TITLE
Dropdown widget does not have a label

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#947]
<img width="377" alt="screen shot 2016-10-28 at 18 04 40" src="https://cloud.githubusercontent.com/assets/13200338/19815085/1983c0b6-9d39-11e6-93aa-5ab2f263538f.png">
